bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/DTOs/AuthenticationMessage.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yioremote.internal.dtos;
```

Comment:
Package names should be all lower case. The directory name is upper case in your case. Also it's good practice to make the names singular. So, `dto` would be good.

Reply:
solved i pushed it first with DTOs than changed it to dtos but this was an error during pushung therefore put dto inside